### ISSUE TITLE
New Customer - Currency and Price list vs New Address

### ISSUE THREAD

**Issue by tun-tun-aung** [frappe/erpnext#11266]
If fill Currency and Price list first and add New Address, filled Currency and Price list cleared.
if fill Currency and Price list first Save and then add New Address, new address not link to this new customer.

**Comment by umairsy** [comment]
Couldn't replicate the issue of Address not saving properly. When creating new Address from Customer, then Customer ID is updated correctly in the new address.

Please save Currency/PriceList and Address separately.

If issue still prevails, request you to share the screenshot/gif so that we can try to replicate on the same lines.

**Comment by tun-tun-aung** [comment]
Dear Umair,

May I send this email explanation with screen shot?

[image: Inline image 1]
Above, i've click New Customer.

[image: Inline image 2]
Then I open this recent New customer to fill detail.

[image: Inline image 3]
I've filled, Currency and Price list, then click New Address.


[image: Inline image 4]
There, New address form appear and I've filled and click save.

[image: Inline image 5]
After I click save button on address form, auto come back to Customer form
page.


[image: Inline image 6]
There above screen shot, you will see Currency and Price List i filled
disappear.

Hope this explanation help you clear and able to replicate.

The system version is :

*ERPNext:* v9.1.6 (master)

*Frappe Framework:* v9.1.8 (master)

-----------
Best Regards,
Tun  Tun  Aung
M:+95-9-5500 976
-----------///------------

  C-22, e-Condo, No.4, Zay North St., Dagon TSP., Yangon, 11191, Myanmar.
          Web  :   www.yd-mm.com     |     Email  :   sales@yd-mm.com
  Tel:    +95 - 9 - 730 42 914 ,     +95 - 1 - 376 180 ,     +95 - 9 - 254
201 769
------------------------------------------------------------------------------------------------
